.ci_support/linux_64_.yaml

```diff
@@ -1,15 +1,12 @@
-c_stdlib:
-- sysroot
-c_stdlib_version:
-- '2.12'
 cdt_name:
-- cos6
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
-zip_keys:
-- - c_stdlib_version
-  - cdt_name
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
+tensorflow:
+- '2.16'
```

Comment:
try deleting these two lines. the renderer has been picking things up a little randomly because the regex isn't enitrely correct.

We typically haven't found issues for installed packages.

Reply:
Do you know where this can be fixed?

Reply:
Also happens in https://github.com/conda-forge/tensorflow-datasets-feedstock/pull/23/files/48d1fcf30b60205389b8d229a295b1d550dc10c6..86a2ba2333e5602052de4e4ff44a3ad271f76147

Reply:
look into issues at conda-smithy i think one is opened with regards to this.

Reply:
I confirmed through other feedstocks that this has nothing to do with pinning the tensorflow version. So ignoring.